<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplicationClass</class>
 <widget class="QMainWindow" name="QtWidgetsApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器ip</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器port</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtContent">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>321</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>文本内容</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSend">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>260</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnConnect">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>260</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDisconnect">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtIp">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPort">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>8080</string>
       </property>
      </widget>
      <widget class="QLabel" name="clientStatusLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>421</width>
         <height>16</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{background-color: rgb(255, 0, 0);}</string>
       </property>
       <property name="text">
        <string>未连接</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication.qrc"/>
 </resources>
 <connections/>
</ui>
